pom.xml

```diff
@@ -151,6 +151,26 @@
                         </headerDefinitions>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>xml-maven-plugin</artifactId>
```

Comment:
- [ ] I think that it's better to place this next to `formatter-maven-plugin`.
Because these are formatter plugins gloup.